### ISSUE TITLE
[node] Virtual install events not showing up on intermediary

### ISSUE THREAD

**Issue by rhlsthrm** [counterfactual/monorepo#2515]
We want to be able to detect when a virtual install is going to happen on the intermediary side and do things pre-install, possibly even block the install. We were able to see PROPOSE and INSTALL events previously, but they seem to have stopped working.

**Comment by snario** [comment]
Will look into this.

**Comment by snario** [comment]
To confirm, you would like to be able to specify a whitelist at runtime of the `Node` of which `xpubs` you do not want to be an intermediary for virtual app installations?

**Comment by rhlsthrm** [comment]
Yes, but more generically we would like control to be able to reject the intermediary install for any reason at all (i.e. if we don't want to allocate the collateral for whatever reason). Previously we were able to see the proposal event and use reject install.
